src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactoryCreateOptions.cs

```diff
@@ -35,10 +35,10 @@ public JsonSerializerOptions SerializerOptions
     /// <summary>
     /// Gets or sets the <see cref="AIJsonSchemaCreateOptions"/> governing the generation of JSON schemas for the function.
     /// </summary>
-    public AIJsonSchemaCreateOptions SchemaCreateOptions
+    public AIJsonSchemaCreateOptions JsonSchemaCreateOptions
```

Comment:
Would it make sense at this point for JsonSchemaCreateOptions and SerializerOptions to be nullable, and then just substitute the default values if they're null when actually constructing the AIFunction? It's not clear to me at this point why that's how Name/Description/AdditionalProperties work but not how these work.

Reply:
We could do that. Making JSO non-nullable specifically is a holdover from when the factory APIs not accepting an explicit JSO were being marked RUC/RDC.